deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -930,6 +935,21 @@ ColorMaskState &ColorMaskState::operator=(const ColorMaskState &colorMaskState)
 std::pair<Local<Object>, Local<FunctionTemplate>> WebGLRenderingContext::Initialize(Isolate *isolate) {
   // Nan::EscapableHandleScope scope;
 
+#if defined(ANDROID) || defined(LUMIN)
+  glFramebufferTextureMultiviewOVRExt = (PFNGLFRAMEBUFFERTEXTUREMULTIVIEWOVR)eglGetProcAddress("glFramebufferTextureMultiviewOVR");
+  if (!glFramebufferTextureMultiviewOVRExt) {
+      std::cerr << "Can not get proc address for glFramebufferTextureMultiviewOVR." << std::endl;
+      sleep(1);
```

Comment:
There is probably no need to sleep, or perhaps even abort. I think as long as we initialize to `nullptr` and log the warning, it would be fine to continue -- only applications that use multiview on ANDROID would be affected by it being missing.

Reply:
8ae3505